superset-frontend/src/explore/components/controls/DndColumnSelectControl/ColumnSelectPopover.tsx

```diff
@@ -117,6 +118,18 @@ const ColumnSelectPopover = ({
     ColumnMeta | undefined
   >(initialSimpleColumn);
 
+  const [width, setWidth] = useState(POPOVER_INITIAL_WIDTH);
+  const [height, setHeight] = useState(POPOVER_INITIAL_HEIGHT);
```

Comment:
What do you think about moving `width/height` state into `useResizeButton` and having the hook return `{resizeButton, width, height}`?

Reply:
Nice, that really put the icing on the cake 🍰